client/components/account-status/style.scss

```diff
@@ -44,20 +45,26 @@
 		}
 	}
 
-	div {
+	.item-label,
+	.item-value {
 		flex-shrink: 1;
 		flex-basis: auto;
 		width: min-content;
 	}
 
-	// label
-	div:first-child {
+	.item-label {
 		color: $gray-900;
 		min-width: 130px;
+		white-space: nowrap;
 	}
 
-	// value
-	div:last-child {
+	@include breakpoint( '<480px' ) {
+		.item-label {
+			min-width: 80px;
```

Comment:
Where are those `80px` coming from? When previewing in iPhone 6/7/8 mode (375px screen width), the second "column" is not properly aligned.

<img width="358" alt="image" src="https://user-images.githubusercontent.com/5311119/120796817-f17e5080-c543-11eb-8e0a-076fff24f6d6.png">

Reply:
It is an empirical value based on the design @LevinMedia referenced in #1464 for smaller screens. For that design "Connected email" label was wider than others and would allow long emails to go on the next line.

https://github.com/Automattic/woocommerce-payments/pull/1464#discussion_r636515619